packages/hathor-rpc-handler/src/rpcHandler/index.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  GetAddressRpcRequest,
+  GetBalanceRpcRequest,
+  GetConnectedNetworkRpcRequest,
+  GetUtxosRpcRequest,
+  PromptHandler,
+  RpcMethods,
+  RpcRequest,
+  SendTxRpcRequest,
+  SignWithAddressRpcRequest,
+} from '../types';
+import { signWithAddress } from '../rpcMethods/signWithAddress';
+import { HathorWallet } from '@hathor/wallet-lib';
+import { getAddress, getBalance, getUtxos } from '../rpcMethods';
+import { getConnectedNetwork } from '../rpcMethods/getConnectedNetwork';
+import { sendTx } from '../rpcMethods/sendTx';
+import { InvalidRpcMethod } from '../errors';
+
+export const handleRpcRequest = async (
+  request: RpcRequest,
+  wallet: HathorWallet,
+  promptHandler: PromptHandler,
```

Comment:
No because the idea is to have a single `promptHandler` that will handle multiple rpc calls, you can see it implemented in the wallet-mobile PR:
https://github.com/HathorNetwork/hathor-wallet-mobile/blob/72be982c10c6b4253b35c795ef607de4d957f6c9/src/sagas/walletConnect.js#L277